person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @id = id
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] This method suppose to be a private method, not public as you did.
[Hint] To make it private the best practice is to put it at the end of the `Person` class and put the word `private` before it like that:
```ruby
private 
  
def of_age?
   @age >= 18
end
```